src/elements.jl

```diff
@@ -69,9 +99,10 @@ local_potential_real(el::ElementCoulomb, r::Real) = -el.Z / r
 
 
 struct ElementPsp <: Element
-    Z::Int  # Nuclear charge
-    symbol  # Element symbol
-    psp     # Pseudopotential data structure
+    Z::Int         # Nuclear charge
+    symbol         # Element symbol
+    psp            # Pseudopotential data structure
+    use_nlcc::Bool # Flag to enable/disable non-linear core correction
```

Comment:
Hm. Probably it shouldn't be at that level, but rather on the Xc term itself, no? Psp by itself should not want to have their nlcc disabled.

Reply:
This was @mfherbst 's suggestion, I used to have it on the Xc term. Not sure what's really best.

Reply:
Terms is where we customize the behavior of how each term behavior, psp is more of a straight read of the data structure and probably shouldn't "hide" user options